src/sonic-py-common/sonic_py_common/util.py

```diff
@@ -0,0 +1,67 @@
+"""
+Syslog and daemon script utility library.
+"""
+
+from __future__ import print_function
+import json
+import logging
+import logging.config
+import sys
+from getopt import getopt
+
+
+# TODO: move to dbsync project.
```

Comment:
Can't move to dbsync, because currently snmp agent also using this:
https://github.com/Azure/sonic-snmpagent/blob/5f8f475b54fbe12b959258bb5567f71577aff0ea/src/sonic_ax_impl/__main__.py